renovate.json

```diff
@@ -15,5 +15,8 @@
             ],
             "versioningTemplate": "docker"
         }
-    ]
+    ],
+    "tekton": {
+        "automerge": true
```

Comment:
Do we want to enable more automerges?

Reply:
But I would like to see when they are coming. I mean, tekton, we can automerge as they come, because it does not trigger a push event. But a base image update, or npm updates etc, we will need to see if the merge happens on a timeframe where we can release or if renovate needs more tweaks to delay the automerge :)